user-manager/src/main/resources/application.yaml

```diff
@@ -0,0 +1,15 @@
+spring:
+  flyway:
+    schemas: users
+  jpa:
+    show-sql: true
+    properties:
+      hibernate:
+        dialect: org.hibernate.dialect.PostgreSQLDialect
+        format_sql: true
+        default_schema: users
+    hibernate:
+      ddl-auto: update
```

Comment:
you use flyway for migration, so you don't need to migrate data again using hibernate.
Advantage of flyway is to store logs of migrations and if we something bad happened we can rollback any time. There's a special table in your db "flyway history" as I remember. 
Right now, you use both flyway and hibernate. Instead, it's better to use hibernate ddl to validate correctness of schema `spring.jpa.hibernate.ddl-auto=validate`
https://rieckpil.de/howto-best-practices-for-flyway-and-hibernate-with-spring-boot/
https://www.baeldung.com/database-migrations-with-flyway